libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/ContextualizedLogger.scala

```diff
@@ -92,4 +129,7 @@ final class ContextualizedLogger private (val withoutContext: Logger) {
       }
     }
 
+  def logTermination[T](implicit loggingContext: LoggingContext): GraphStage[FlowShape[T, T]] =
+    new ContextualizedLogger.TerminationLogger[T](info)
```

Comment:
Why are we logging this at info? All stream closures will log then something, no? What is the purpose of this logging?

Reply:
We are logging at the same level as the stream creation